lib/src/accordion/Accordion.test.js

```diff
@@ -5,23 +5,24 @@ import DxcAccordion from "./Accordion";
 describe("Accordion component tests", () => {
   test("Accordion renders with correct text", () => {
     const onChange = jest.fn();
-    const { getByText } = render(<DxcAccordion label="Accordion" onChange={onChange} />);
-
+    const { getByText } = render(<DxcAccordion label="Accordion" assistiveText="Assistive text" onChange={onChange} />);
     expect(getByText("Accordion")).toBeTruthy();
+    expect(getByText("Assistive text")).toBeTruthy();
+  });
+  test("Accordion renders expanded by default when it is uncontrolled", () => {
+    const { getByRole } = render(<DxcAccordion label="Accordion" defaultIsExpanded><div>test-expanded</div></DxcAccordion>);
+    const accordion = getByRole("button");
+    expect(accordion.getAttribute("aria-expanded")).toBe("true");
   });
-
   test("Calls correct function on click", () => {
     const onChange = jest.fn();
     const { getByText } = render(<DxcAccordion label="Accordion" onChange={onChange} />);
-
     fireEvent.click(getByText("Accordion"));
     expect(onChange).toHaveBeenCalled();
   });
-
   test("Controlled accordion", () => {
     const onChange = jest.fn();
     const { getByText } = render(<DxcAccordion label="Accordion" onChange={onChange} isExpanded={true} />);
-
     fireEvent.click(getByText("Accordion"));
```

Comment:
We should add `expect(getByRole("button").getAttribute("aria-expanded")).toBe("true");` to this test.

Reply:
Agree!